### PR TITLE
Sync to dev

### PR DESCRIPTION
**Pull request by luiskuhn**
Sync this baby to dev :)

### CODE REVIEW

**Review by Steffengreiner — CHANGES_REQUESTED** [review]
Heya @luiskuhn nice work 👍 
However I'd advice against merging the `master` branch into `development `directly, since the `hotfix` is the only changes that happened it would make more sense to merge the `hotfix `branch into `development.` 
You could also think about bumping the version on the `hotfix `branch with a minor version increase and adding a SNAPSHOT tag (Also adding a seperate section to the changelog) to indicate that the version on `development` is a snapshot.
I made a similiar PR for the [barcode-dragon-portlet](https://github.com/qbicsoftware/barcode-dragon-portlet/pull/35/files)

**Comment by luiskuhn** [comment]
Makes sense, I'm on it

File: docs/conf.py [hunk 1]
```diff
@@ -55,9 +55,9 @@
 # the built documents.
 #
 # The short X.Y version.
-version = '1.3.1'
+version = '1.3.5'
```
Comment (jenniferboedker) [inline docs/conf.py:58]:
```suggestion
version = '1.3.6'
```

File: docs/conf.py [hunk 2]
```diff
@@ -55,9 +55,9 @@
...
 # The full version, including alpha/beta/rc tags.
-release = '1.3.1'
+release = '1.3.5'
```
Comment (jenniferboedker) [inline docs/conf.py:60]:
```suggestion
release = '1.3.6'
```

**Pull request merged**